### PR TITLE
Enhance the saved RPC methods screen

### PR DESCRIPTION
**Pull request by jeesunikim**
saved-requests-rpc-methods

### CODE REVIEW

File: src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx [hunk 1]
```diff
@@ -33,19 +37,36 @@ export const SavedEndpointsPage = () => {
   const { saved, updateSavedActiveTab, setParams } = endpoints;
   const router = useRouter();
 
-  const [savedEndpoints, setSavedEndpoints] = useState<SavedEndpointHorizon[]>(
```
Comment (jeesunikim) [inline src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:36]: renamed to `savedEndpointsHorizon` and `setSavedEndpointsHorizon `

File: src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx [hunk 2]
```diff
@@ -33,19 +37,36 @@ export const SavedEndpointsPage = () => {
...
-    [],
-  );
+  const [savedEndpointsHorizon, setSavedEndpointsHorizon] = useState<
+    SavedEndpointHorizon[]
+  >([]);
+  const [savedRpcMethods, setSavedRpcMethods] = useState<SavedRpcMethod[]>([]);
+  const [expandedPayloadIndex, setExpandedPayloadIndex] = useState<{
```
Comment (jeesunikim) [inline src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:44]: tracking `view payload` for each saved rpc method

File: src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx [hunk 3]
```diff
@@ -186,7 +210,130 @@ export const SavedEndpointsPage = () => {
...
+                />
+              </div>
+              <Box
+                gap="lg"
+                direction="row"
+                align="center"
+                justify="space-between"
+                addlClassName="Endpoints__urlBar__footer"
+              >
+                <Box gap="sm" direction="row">
```
Comment (quietbits) [inline src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:255]: We'll need to add a `wrap` prop to this `Box` to wrap buttons if they don't fit on smaller screens. This prop was added in [my branch](https://github.com/stellar/laboratory/blob/sign-txn-updated-ux/src/components/layout/Box/index.tsx#L12), which is not merged yet.

Example: set it manually for the first item.

image

**Review by quietbits — APPROVED** [review]

File: src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx (hunk 3 above)
Comment (jeesunikim) [reply src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:255]: i promise i'll check mobile from now on

**Pull request merged**